src/zcc/zcc.c

```diff
@@ -2741,7 +2741,7 @@ int 	find_zcc_config_fileFile(const char *program, char *arg, int gc, char *buf,
 		*/
 		return (gc);
 	} 
-	// Without a config file, we should just exit
+	// Without a config file, we should just print usage and then exit
```

Comment:
I mean, should we `fprintf(stderr, "No config file!\n");` before printing usage?